polaris-react/src/components/ActionList/components/Item/Item.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, {forwardRef, Ref} from 'react';
```

Comment:
```suggestion
import React, {forwardRef} from 'react';
import type {Ref} from 'react';
```

This should fix the lint failure for the node CI checks